sync-pingdom-ec2-security-groups.py

```diff
@@ -120,17 +120,22 @@ def main():
         default='tcp',
         help='The protocol used by the Pingdom probe')
     parser.add_argument(
-        '--port',
+        '--from-port',
         type=int,
         default=80,
-        help='The port on which Pingdom probes')
+        help='The port on which Pingdom probes (start of range)')
+    parser.add_argument(
+        '--to-port',
+        type=int,
+        default=443,
+        help='The port on which Pingdom probes (end of range)')
```

Comment:
This way the script defaults to opening all ports in the range [80, 443]; let's keep the default of only port 80.